### PR TITLE
Switch to ci-install.sh

### PR DESCRIPTION
**Pull request by mcafaro**

### CODE REVIEW

File: tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts [hunk 1]
```diff
@@ -13,11 +13,13 @@ process.env.SYSTEM_SERVERTYPE = "hosted";
 
 tr.registerMock("azure-pipelines-tool-lib/tool", {
     downloadTool(url: string) {
-        if (url !== "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh"
-            && url !== "https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh") {
+        if (url === "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh") {
+            return "install.sh";
```
Comment (jwpereira) [inline tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts:17]: Given the change in `tasks/install-matlab/v0/main.ts`, are there modes where just `install.sh` would be used and not `ci-install.sh` 🤔

**Review by jwpereira — APPROVED** [review]
Overall looks good to me 😄

File: tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts (hunk 1 above)
Comment (mcafaro) [reply tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts:17]: This is the matlab-deps url, which has kept the install.sh name.

Comment (jwpereira) [reply tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts:17]: Ahhhh oops that makes sense; I forgot about matlab-deps 😂

**Pull request merged**